nextjs/hooks/websockets/index.tsx

```diff
@@ -44,6 +44,11 @@ function useWebsockets({ room, token, permissions, onNewMessage }: Props) {
     return () => {};
   }, []);
 
+  useEffect(() => {
+    channel?.off('new_msg');
+    channel?.on('new_msg', onNewMessage);
+  }, [onNewMessage]);
```

Comment:
@emilos this was the only way I found to be able to get the updated scope from the feed view

Reply:
seems fine for first pass!